setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "😎 Installing project dependencies for"
+npm install
+
+packages="auth backend domain"
+for package in $packages; do
+  cd $package
+  echo "😎 Installing dependencies for '$package'"
+  npm install
+  cd ..
+done
```

Comment:
This is a perfectly valid approach and keeps things simple. Perhaps for the future it might be worth looking into the recently introduced [Serverless Compose](https://www.serverless.com/framework/docs/guides/compose) feature which seems to be the aimed at becoming the new canonical way to deploy multiple services.

Reply:
Oh nice, I did not know about compose! Looks very promising. I am hesitant to use it as part of this PR but definitely something worth looking into for another PR! TNX